app/app.view.ts

```diff
@@ -39,10 +39,18 @@ namespace $.$$ {
 			return formula_html( str )
 		}
 
+		@ $mol_mem_key
+		card_link( n: number ) {
+			let link = this.fetch_by_number( n )?.data[ 0 ]?.attributes?._gnome_material_id
+			return `https://optimade-gnome.odbx.science/v1/structures/data/gnome_data/by_id.zip/data/gnome_data/by_id/${link}.CIF`
+		}
+		
+
 		@ $mol_mem_key
 		card_loaded( n: number ) {
 			try {
 				this.card_name( n )
+				this.card_link( n )
```

Comment:
@luckylionheart please, see https://github.com/mpds-io/this-material-does-not-exist/pull/4/files#r1667184547

Reply:
I solved this problem in next commit.